### DISCUSSION TITLE
How to connect to a MongoDB using a Cypress task

### DISCUSSION THREAD

**Discussion by estefafdez** [cypress-io/cypress#16044] [Questions and Help] [answered]
Hello again! 

I'm looking for a real example of how to connect to a MongoDB with cypress using a task. 

I've tried to find information about it but the only related issue I could found is: https://github.com/cypress-io/cypress/issues/2902 which provide information about why you need to use a task but not a real example of how to really do it. 

Is anyone else having the same problem? How do you solve it? I'm looking forward to see the answers from the Community. 

Thanks!

**Comment by bahmutov** [comment, marked as answer]
See https://glebbahmutov.com/blog/testing-mongo-with-cypress/ and https://github.com/bahmutov/cypress-example-mongodb
